examples/InputWithDecorations.advanced.tsx

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import {
+  InputWithDecorations,
+  Icon,
+  Text,
+  Divider,
+} from '@itwin/itwinui-react';
+import {
+  SvgCaretUpSmall,
+  SvgCaretDownSmall,
+  SvgSearch,
+} from '@itwin/itwinui-icons-react';
+
+export default () => {
+  return (
+    <InputWithDecorations>
+      <Icon padded>
+        <SvgSearch />
+      </Icon>
+      <InputWithDecorations.Input placeholder='Search...' />
+      <Text style={{ paddingInline: 8 }}>0 / 5</Text>
```

Comment:
this inline padding always bothers me

Reply:
Any suggestions how to fix this to now bother you?